website/blog/2025-05-21-podman-ai-lab-openvino.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Enabling OpenVINO Inference in Podman AI Lab
+description: Use Podman AI Lab to run OpenVINO inference workloads locally.
+authors: [jeffmaury]
+tags: [podman-desktop, podman, extension, ai, llm, local, podman-ai-lab, openvino]
+hide_table_of_contents: false
+---
+
+# Enabling OpenVINO Inference in Podman AI Lab
+
+## Introduction to Podman AI Lab
+
+Podman AI Lab is an open-source platform designed to simplify the deployment, management, and experimentation of AI workloads using container technology. It provides a user-friendly interface for running, testing, and scaling AI models locally or in the cloud, leveraging the power and flexibility of Podman containers.
+
+## What is OpenVINO?
+
+OpenVINO™ (Open Visual Inference and Neural Network Optimization) is an open-source toolkit developed by Intel to accelerate AI inference on a variety of hardware, including CPUs, GPUs, and specialized accelerators. It optimizes deep learning models for fast, efficient inference, making it a popular choice for edge and cloud AI applications.
+
+## Using OpenVINO in Podman AI Lab
+
+Podman AI Lab now supports OpenVINO as an inference provider. This means you can:
+
+- Select an OpenVINO compatible model when starting an inference server or playground.
```

Comment:
Should we mention how a user would know the model is "OpenVINO compatible"? Do we have a tag they can filter on?